judging/generate_predictions_local.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
Copied over from judging repo. We can delete it from there once we're happy with this solution.